CMakeLists.txt

```
@@ -19,6 +20,7 @@ elseif(CMAKE_SIZEOF_VOID_P EQUAL 4)
   message(STATUS "Building for 32 bit platform")
 endif()
 
+option(ENABLE_STANDALOND_WINTLS "Enable Standalone WINTLS" OFF)
```

```suggestion
option(ENABLE_STANDALONE_WINTLS "Enable Standalone WINTLS" OFF)
```

There's a typo here. I don't think there's any naming standard for CMake options, however I often see the options prefixed by the project name (like for #defines). So it could be something like `WINTLS_STANDALONE_ASIO` here.